org-cyf-tracks/content/business-analysis/prep/index.md

```diff
@@ -0,0 +1,11 @@
++++
+title = "prep"
+description = "This is the use case you will use throught this track."
+layout = "prep"
+emoji = "📅"
+menu_level = ["sprint"]
```

Comment:
To have this show up on the track contents, it needs to be in the module-level menu:

```suggestion
menu_level = ["module"]
```